holochain-introduction-for-developers/holochain-brief-introduction.md

```diff
@@ -0,0 +1,75 @@
+# What is Holochain?
+
+> Holochain is a _protocol_ and _framework_ for creating cryptographically secure, portable, and distributed applications. It provides the necessary tools to allow individuals to maintain control over their identity and data, while interacting safely with a network of untrusted peers.
+>
+> The purpose of this document is to help developers and technologists quickly understand:
+>
+> * Why we're building Holochain
+> * What the project consists of
+> * What advantages it has over server-based and blockchain-based architectures
+
+## The problem
+
+The Internet is decentralized by design. Many services, such as the web and email, have no central points. But in order to move through the Internet confidently, we need **trust.** How can you know I am who I say I am, and how can you be sure that I'm acting with integrity?
```

Comment:
Um, only parts of the internet, especially routing.  Other parts are quite centralized.  DNS is centralized (how many top-level dns servers are there, 12?) IP address allocation is centralized.  And it's architecture yields centralizing pareto effects as you note.  The problem is that the Internet isn't _fully_ decentralized.   Maybe the initial sentence wouldn't have got me if it were more like:  "We often think that the internet is decentralized by design... "

Reply:
okay, point noted. I guess that just saying 'The Internet' is a bit vague, because while the routing layer is decentralisation-friendly, the systems that support it are emphatically not. I'll chew on this.

Reply:
changed some of the language here; the new diff will show it.